packages/webpack/mixins/build/mixin.core.js

```diff
@@ -79,6 +80,12 @@ class WebpackBuildMixin extends Mixin {
       })
     );
   }
+  runChecks() {
+    const { _workspace } = this.config;
+    return detectDuplicates(_workspace, 'webpack').map(
+      (duplicate) => `package '${duplicate}' should be installed just once`
```

Comment:
I thought about that, too - I ended up with what is there now because I felt it would be better to leave it up to the individual packages to run these kinds of specific checks.

Additionally, I would have found it strange to add a config field for this.

Reply:
Thought some more: I really do not want to add a specific API or config for this.

While unfortunately rather common, this is a very narrow kind of issue - and we are already going above and beyond what can reasonably be expected with this check.